### DISCUSSION TITLE
config.json File

### DISCUSSION THREAD

**Discussion by vtelco** [rtbrick/bngblaster#243] [Q&A] [answered]
Hi guys, I tried to start a setup using BNG Blaster but I didn't find this file (config.json) after installation.

vtelco@bng-blaster:~/bngblaster/build$ make all
Consolidate compiler generated dependencies of target test-utils
[  2%] Built target test-utils
Consolidate compiler generated dependencies of target test-checksum
[  4%] Built target test-checksum
[  7%] Built target lwipcontribportunix
[ 51%] Built target lwipcore
[ 88%] Built target bngblaster
Consolidate compiler generated dependencies of target test-protocols
[ 89%] Built target test-protocols
Consolidate compiler generated dependencies of target test-decode-pcap
[ 90%] Built target test-decode-pcap
[100%] Built target lspgen
vtelco@bng-blaster:~/bngblaster/build$ make test
Running tests...
Test project /home/vtelco/bngblaster/build
    Start 1: TestUtils
1/3 Test #1: TestUtils ........................   Passed    0.00 sec
    Start 2: TestChecksum
2/3 Test #2: TestChecksum .....................   Passed    0.00 sec
    Start 3: TestProtocols
3/3 Test #3: TestProtocols ....................   Passed    0.00 sec

100% tests passed, 0 tests failed out of 3

Total Test time (real) =   0.00 sec

vtelco@bng-blaster:~/bngblaster/build$ sudo bngblaster -C config.json -I
JSON config error: File config.json Line -1: unable to open config.json: No such file or directory
Error: Failed to load configuration file config.json

root@bng-blaster:~# find / -name config.json
root@bng-blaster:~#

Could you help me ?

**Comment by GIC-de** [comment, marked as answer]
The file config.json is just a placeholder for your actual configuration. You can fin some fully working configuration example in the Quick Start Guide, which I recommend to start with. 

https://rtbrick.github.io/bngblaster/quickstart.html
